manager/src/main/java/com/redhat/service/bridge/manager/BridgesServiceImpl.java

```diff
@@ -73,6 +68,10 @@ public Bridge createBridge(String customerId, BridgeRequest bridgeRequest) {
 
     @Transactional
     protected Bridge doCreateBridge(String customerId, BridgeRequest bridgeRequest) {
+        if (bridgeDAO.findByNameAndCustomerId(bridgeRequest.getName(), customerId) != null) {
```

Comment:
I'm not sure this changes anything. 

First `createBridge` created a transaction and `doCreateBridge` attached to the previous one (the standard behaviour for `REQUIRED`) and now there's no transaction outside and these lines are moved inside `doCreateBridge`. 

It's still one transaction and the same one

Reply:
The call to schedule work is no longer inside the transaction and the code _structure_ is identical `deleteBridge(..)`, `createProcessor(..)` and `deleteProcessor(..)`. @robpblake has already proposed we can now dispense with the separate `X` and `doX` methods _consistently_ across both bridges and processors. I'd prefer to create another JIRA to handle the change in one go than pollute this PR (for removal of the `Work`). WDYT?

Reply:
I'm not sure you get what you mean, but it should be fine anyway